docs/0.2/references/product/additional_attributes.yaml

```diff
@@ -0,0 +1,5 @@
+- product_namespace: GS1
```

Comment:
This should include the GDSN_3_1 field. I know schema has the quirk that you can't redefine an attribute but for this case I dont think youd be changing this on the same circuit. Probably something we could clean up for Schema.